general_rules/Procedure.tex

```diff
@@ -187,23 +172,6 @@ \subsection{Operators}
 Penalty may apply when using a custom operator.
 
 
-\subsection{Moderator}
```

Comment:
So do we eliminate the usage of any other language than English?

Reply:
No, that is not what this rule says. This rule is meant to have someone of the team be available to provide live commentary to the audience. Which is not/rarely organised by the LOC.

Reply:
Oh sorry didn't read carefully enough